Automattic/jetpack
# Monitor: Don't query last down time

We haven't been showing the "Last Downtime" data for Monitor since 0be8681b9fcfc602429b79b51177a8057da9c8c4

This removes the WP-API call (and subsequent xmlrpc request to dotcom).  It will have two positive effects: 

1. Faster admin page load 
2. Not as many random instances with the xmlrpc request failing with a 500 error.  I've see this come up a bit with our support.  

To test: 
- Make sure that we're not making any calls to `/wp-json/jetpack/v4/module/monitor` when viewing the Dashboard
- Make sure activating/deactivating Monitor still works

Will also fix #5208
